test-cypress/support/repository-commands.js

```diff
@@ -0,0 +1,22 @@
+import repoTemplate from '../fixtures/repo-template.json';
+import _ from 'lodash';
+
+const REPOSITORIES_URL = '/rest/repositories/';
+
+Cypress.Commands.add('createRepository', (options = {}) => {
+    cy.request({
+        method: 'POST',
+        url: REPOSITORIES_URL,
+        body: _.defaultsDeep(options, repoTemplate),
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    }).should((response) => expect(response.status).to.equal(201)); // 201 Created
+});
+
+Cypress.Commands.add('deleteRepository', (id) => {
+    const url = REPOSITORIES_URL + id;
+    cy.request('DELETE', url).should((response) => expect(response.status).to.equal(200));
+    // TODO: needed?
+    cy.request({method: 'HEAD', url, failOnStatusCode: false}).should((response) => expect(response.status).to.equal(404));
```

Comment:
Create specification that test these two and move the HEAD check there. No need to check every time.